scripts/always.sh

```diff
@@ -3,6 +3,9 @@
 echo "starting httpd"
 systemctl start httpd.service
 
+echo "turning off firewalld"
+systemctl stop firewalld
```

Comment:
wouldn't it make sense to open ports as and when necessary for each service, rather than switching off the firewall altogether?

on a secured, production environment, you wouldn't do this... would you?